orchprovider/k8s/v1/util_test.go

```diff
@@ -137,3 +137,26 @@ func TestK8sUtilServices(t *testing.T) {
 		}
 	}
 }
+
+// TestGetOutClusterCS tests the working of TestgetOutClusterCS() method.
+func TestGetOutClusterCS(t *testing.T) {
+	// a noop pvc that in turn signals use of defaults
+	pvc := &v1.Volume{
+		Namespace: "default",
+	}
+	//pvc.Labels = map[string]string{}
```

Comment:
remove this commented code.

Reply:
ok sir

Reply:
should i raise a pr once again ?

Reply:
you can commit the new changes here again.